src/RefractAPI.cc

```diff
@@ -39,6 +41,18 @@ namespace drafter {
             }
         }
 
+        refract::ArrayElement* CreateArrayElement(refract::IElement* value, bool rFull = false)
```

Comment:
@klokane I think Pavan was saying you could add the `bFull = false` parameter to the existing `CreateArrayElement` function.